src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java

```diff
@@ -114,6 +115,9 @@ public ParseStatus parseJavadocAsDetailNode(DetailAST javadocCommentAst) {
             final ParseTree parseTree = parseJavadocAsParseTree(javadocComment);
 
             final DetailNode tree = convertParseTreeToDetailNode(parseTree);
+            // Column number of COMMENT_CONTENT + 1 for leading asterisk
```

Comment:
:), if I do a comment I expect reply on it.
We do not need extra code .. the same for comments.